wallet/res/layout/activity_search_dashpay_profile_1.xml

```diff
@@ -5,6 +5,7 @@
     android:id="@+id/root"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:focusableInTouchMode="true"
```

Comment:
This is added because the SearchUsernameActivity was giving focus to the Search TextView and bypassed the transition.